edit/management.cattle.io.project.vue

```diff
@@ -112,11 +126,20 @@ export default {
   methods: {
     async save(saveCb) {
       try {
-        // clear up of the unused resourceQuotas will now be done on the model side
-        const savedProject = await this.value.save();
+        // we allow users with permissions for projectroletemplatebindings to be able to manage members on projects
+        if (this.canManageProject) {
+          // clear up of the unused resourceQuotas will now be done on the model side
+          await this.value.save();
+        }
 
         if (this.membershipUpdate.save) {
-          this.membershipUpdate.save(savedProject.id);
+          let projId = this.value.id;
+
+          if (projId.includes('local/')) {
+            projId = projId.replace('local/', 'local:');
```

Comment:
This looks like it's converting the steve id to a norman one, if so should it be for all clusters and not just local?

The same thing happens in the model for the project, see normanEditProject. Might be worth adding a `get normanId` there and use it in both places

Reply:
this.val.norman.id will get you the correct id